src/client/Lib.cpp

```diff
@@ -150,6 +151,7 @@ void __attribute__((constructor)) tazerInit(void) {
         timer.start();
         Loggable::mtx_cout = new std::mutex();
         InputFile::_cache = new Cache(BASECACHENAME, CacheType::base);
+        LocalFile::_cache = new Cache(BASECACHENAME, CacheType::base);
```

Comment:
We might want to think about the implications of creating two separate caches...
do we want to allow both input and local at the same time?
thinking about it, can local just collapse down to inputfile just using "localfilecache/urlcache" as the llc?